test/custom-service-client-options/generated/Xml/README.md

```diff
@@ -7,26 +7,35 @@ This package contains an isomorphic SDK (runs both in node.js and in browsers) f
 - Node.js version 8.x.x or higher
 - Browser JavaScript
 
+### Prerequisites
+
+You must have an [Azure subscription](https://azure.microsoft.com/free/).
+
 ### How to install
 
 To use this SDK in your project, you will need to install two packages.
 - this package that contains the client
-- `@azure/identity` that contains different credentials for you to authenticate the client using Azure Active Directory.
+- `@azure/identity` that provides different mechanisms for the client to authenticate your requests using Azure Active Directory.
 
-Install both packages using the below commands.
+Install both packages using the below command:
 ```bash
-install the package using the `npm install` command
-npm install @azure/identity
+npm install --save <package name> @azure/identity

```

Comment:
Is there any way to add the actual package name?

Reply:
it is included if it is defined. See the example I have in the PR description.